### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by vividos**
### Description of Change ###

Here's a PR, doing a little housekeeping, like updating the NuGet packages and cleaning up the sample code. Most changes are in the Samples project. The actual plugin project file was revised a bit, and the Microsoft.SourceLink.GitHub package was updated to the latest version.

### Issues Resolved ### 

- none

### Platforms Affected ### 

- none

### CODE REVIEW

**Comment by vividos** [comment]
@jfversluis I have some more PRs in the pipeline that I think we should do a stable version with this version, and after that increase the plugin version to 2.2.x-beta.

File: samples/Plugin.FilePicker.Sample.Forms/Droid/Plugin.FilePicker.Sample.Forms.Droid.csproj [hunk 1]
```diff
@@ -76,7 +76,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Xamarin.Forms">
-      <Version>3.3.0.912540</Version>
+      <Version>4.5.0.356</Version>
```
Comment (jfversluis) [inline samples/Plugin.FilePicker.Sample.Forms/Droid/Plugin.FilePicker.Sample.Forms.Droid.csproj:79]: Is this absolutely necessary? The lower the better, now we are forcing people to upgrade to 4.5 when they want to upgrade this

Comment (vividos) [reply samples/Plugin.FilePicker.Sample.Forms/Droid/Plugin.FilePicker.Sample.Forms.Droid.csproj:79]: It's only in the sample projects. The Plugin.FilePicker only references the SourceLink NuGet package.

Comment (jfversluis) [reply samples/Plugin.FilePicker.Sample.Forms/Droid/Plugin.FilePicker.Sample.Forms.Droid.csproj:79]: Oooo right 😴

**Comment by jfversluis** [comment]
Ugh. Stuff with the pipeline going on. I'll cheat this one in, it should be good for the next PR

**Pull request merged**